samples/kubeflow/README.md

```diff
@@ -11,52 +12,91 @@ This demo uses the default stack:
 
 ## Actors and flows
 
-A _data owner_ uploads a transactions.csv file to an s3 compatible object storage (e.g., Ceph, minio, IBM Cloud Object Storage, Amazon S3, etc.). Then, the data owner registers the asset in Egeria data catalog and adds tags to the asset to mark it as finance data. Finally, the data owner registers the credentials to access this asset in Vault credentials manager.
+A _data owner_ uploads a transactions.csv file to as s3 compatible object storage (e.g., Ceph, minio, IBM Cloud Object Storage, Amazon S3, etc.). Then, the data owner registers the asset in Egeria data catalog and adds tags to the asset to mark it as finance data. Finally, the data owner registers the credentials to access this asset in Vault credentials manager.
 
 A _data steward_ creates policies in OPA policy manager. These policies set transformations to perform on data columns when accessed for the purpose of fraud-detection.
 
 A _data user_ creates a notebook server in Kubeflow and a notebook with the business logic for creating a fraud detection model. The data user also creates a `M4DApplication` resource that references the transactions.csv asset from the data catalog. This allows the code in the notebook to load the data (after policies are applied to it).
 
 ## Installation
 
-Run the following script to install The Mesh for Data and all dependencies required for this demo. It will take a while for all containers to be ready due to image downloads.
+Run the following script to install The Mesh for Data core components and third party dependencies required for this demo.
+It will take a while for all containers to be ready due to image downloads. 
 
+```bash
+kubectl config set-context --current --namespace=m4d-system
+
+# For running on OpenShift
+WITHOUT_OPENSHIFT=false ./hack/install.sh
 
+# For running on Kind
+./hack/install.sh
+```
+
+Install kubeflow on your cluster. This sample (and code) was tasted against kubeflow v1.0.2.
+On a Kind cluster for example you can use the following script to install it:
 ```bash
-kind create cluster
-./install/install.sh
+cd samples/kubeflow/install/kubeflow
+./install_kubeflow.sh
+
+cd -
+```
+
+Install the Arrow-Flight module
+```
+kc apply -f https://raw.githubusercontent.com/IBM/the-mesh-for-data-flight-module/master/module.yaml
 ```
 
 ## Data owner instructions
 
-We already have transactions.csv stored in COS (TODO: don't assume that).
-Register the asset using the following command:
+Upload the file `data.csv` to an object-storage of your choice. For example, we tested the code against IBM's COS.
+`data.csv` contains the first 100 rows from the following [data set](https://www.kaggle.com/ntnu-testimon/paysim1/data)
+
+After uploading the data to some object-storage, alter the `fullPath` field in the json `example_transactions.csv.json` to contain the details of your object storage location, etc.
```

Comment:
After uploading the data to some object-storage ---> After uploading the data to an  object store

Reply:
Fixed